src/Ledger/Foreign/HSLedger.agda

```diff
@@ -156,11 +156,12 @@ HsGovParams : GovParams
 HsGovParams = record
```

Comment:
Why is `HsGovParams` not called `HSGovParams`?

None of the other types starts with `Hs` (i.e. `H` and *lower-case* `s`).

Reply:
I didn't touch that part in this PR. Since we'll probably be having a bunch of changes in that file soon I'm not going to bother to fix that inconsistency now.